server/conf/sites.yaml

```diff
@@ -29,3 +29,8 @@ sites:
 # the_blaze: https://www.theblaze.com
 # slate: https://slate.com
 # bloomberg: https://www.bloomberg.com
+# latimes: https://wwww.latimes.com
+# bbc: https://www.bbc.com
+# npr: https://wwww.npr.org
```

Comment:
`https://www.npr.org` and same for latimes. Also, can you add the corporation data for these sites?